### PR TITLE
Add bulk update example

### PR DESCRIPTION
**Pull request by jaen**
Adds an example showing how one can use the Migration API to perform bulk updates.

Also addresses a few clarity/wording comments I've received.

### CODE REVIEW

File: src/examples/update/README.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+> getInBatches = async function* (parameters = {}) {
+    // We start with the first page (or a page caller specified)
+    let page = parameters.page || 1;
+    let nextPage = true;
+
+    // We will loop while there is a next page of document results
+    // When the function will finish looping, the generator will complete as well
+    while (nextPage) {
+      // Override the page parameter to the one we are at, pass all other parameters
+      // through, to support
```
Comment (lihbr) [inline src/examples/update/README.md]: I think you're missing the end of the sentence here, no?

**Review by lihbr — APPROVED** [review]
Awesome work! Just on small thing perhaps but feel free to merge when all good for you ☺️

---

_Also, not a result of this PR, but I noticed there's both a `package-lock.json` and a `yarn.lock` file. I don't know how you feel about it but I'd delete the `yarn.lock` file (since yarn can also honor the `package-lock.json` if present)_

File: src/examples/update/README.md (hunk 1 above)
Comment (jaen) [reply src/examples/update/README.md]: Yes, slipped my mind; finished it.

**Comment by jaen** [comment]
@lihbr re: lockfiles, as we discussed `yarn install` is supposed to be primary way of managing dependencies, with `package-lock.json` synced using `yarb lockfile:sync`. We can revisit this in the future if we want to simplify it by providing only one option.

**Review by lihbr — APPROVED** [review]

**Pull request merged**